homeassistant/components/raspberrypi/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "raspberrypi",
```

Comment:
We normally separate words with underscore in domain names. The mark is Raspberry Pi.